en/services/component_information.md

```diff
@@ -179,18 +180,75 @@ Specifically, the following information is provided:
 
 A GCS can provide a UI for testing outputs based on the configured output functions, by iterating over all output channels and collecting the configured actuator output functions, and then utilizing the `MAV_CMD_ACTUATOR_TEST` command.
 
+## Translation
 
-## Open Issues
+At high-level, metadata translation works as following:
+
+- The metadata provider sets the `translationUri` in [general metadata file](#COMP_METADATA_TYPE_GENERAL) for each metadata type that supports translation.
+  There is no CRC for the resource pointed to by this URI, as translations might change independently of metadata (for example, adding or changing translations for a particular language, or adding translations for a new language).
+- The `translationUri` URL points to a summary JSON file that contains links to the separate files that contain each translation of the particular metadata type.
+  The translation summary JSON file also contains modification timestamps for each linked translation file so that a GCS can determine whether the file has been updated.
+  The translation files are a (compressed) file in [TS file format](https://doc.qt.io/qt-6/linguist-ts-file-format.html).
+- A client (GCS) downloads the summary file then uses its contained to locate and download the translation file(s) it is interested in.
+- The client can then apply the downloaded translations to the metadata json file(s) (which contains annotations for which tags to translate) using the translation TS.
```

Comment:
It is not clear exactly what you are saying here by "using the translation TS". Don't the translated files you get provide enough context to create a translated version of the original? 

We could invert all this and say something like:

- The translation system creates a translation source file for each metadata JSON file in TS file format, using the XXX toolchain to extract appropriate tags using annotations in the metadata files.
  The TS file format is a well understood translation format, with good tooling and wide support in translation services.
- This translation file is translated, creating individual files for each language in the TS file format.
- A summary file is created that provides URL links to each of the translation files, along with a modification date for each file.
- The URL is provided to clients in the `translationUri` field in the ...

Just a thought.

Reply:
Yes, correct. The wording is a bit off I guess.
Sure, we can also go with that.

Reply:
I think it's OK as is. I'll just tidy the errors I injected ...
```suggestion
- The client can then apply the downloaded translations to the metadata json file(s) (which contains annotations for which tags to translate).
```